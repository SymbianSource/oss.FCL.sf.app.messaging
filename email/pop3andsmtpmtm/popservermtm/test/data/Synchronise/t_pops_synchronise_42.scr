+OK demon POP3 server ready (demon.co.uk)
+OK username accepted
+OK password accepted
-ERR message already deleted
** Read cancelled
+OK disconnecting
