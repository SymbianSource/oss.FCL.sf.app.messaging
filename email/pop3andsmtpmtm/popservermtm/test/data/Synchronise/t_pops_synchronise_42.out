DELE 2
QUIT
